eoscostarica/eos-rate
# Fix/299 display website text

### GH Issue

#

### Steps to test

1. go to EOS RATE web app
2. go to BPs list page
1. choose one BP and see profile

### Checklist
